project_config/lmic_project_config.h

```diff
@@ -3,7 +3,7 @@
 #define CFG_us915 1
 //#define CFG_au915 1
 //#define CFG_as923 1
-// #define LMIC_COUNTRY_CODE LMIC_COUNTRY_CODE_JP	/* for as923-JP */
+//#define CFG_as923jp 1
```

Comment:
Thanks for your contribution.

I don't understand this part of change. The code on line 6 is the correct way to handle this (and in fact `lmic_confgi_preconditions.h` expands the shorthand). We avoid `#ifdef` as much as possible in the code; the config system converts `#ifdef` to equivalent symbols that are always either 1 or 0.